Задание на спринт 8

Yarogi opened the pull request:
Задание на спринт 8

Yarogi committed:
Задание на спринт 8

doc-ODA commented on app/src/main/java/com/example/playlistmaker/MainActivity.kt:
```diff
@@ -1,11 +1,35 @@
+        }
+        btnSettings.setOnClickListener {
+            val settingsIntent = Intent(this, SettingsActivity::class.java)
+            startActivity(settingsIntent)
```
Привет! Рекомендация. Смотри у тебя часть кода повторяется, как вариант можно сделать функцию, в которую передается class и она будет осуществлять переход на новый экран. Это избавит от повторов кода.

doc-ODA commented on app/src/main/java/com/example/playlistmaker/SettingsActivity.kt:
```diff
@@ -0,0 +1,24 @@
+            //val mainIntent = Intent(this, MainActivity::class.java)
+            //startActivity(mainIntent)
+            //
+            this.finish()
```
Рекомендация. Старайся удалять закомментированный код, он не нужный мусор. Если в команде будет 10 человек и каждый будет оставлять такое, код разрастется и его будет сложно читать.

doc-ODA commented on app/src/main/res/drawable/arrow_back.xml:
```diff
@@ -0,0 +1,9 @@
+    android:viewportHeight="16">
+  <path
+      android:pathData="M16,7H3.83L9.42,1.41L8,0L0,8L8,16L9.41,14.59L3.83,9H16V7Z"
+      android:fillColor="#1A1B22"/>
```
Рекомендация. Цвета картинок тоже можно в ресурсы вынести.

Yarogi committed:
Изменены отступы в соответсвии в сторогом соответсвии с шалоном.
На экранах заглушках заголовки формы приведены к общему стандарту.
Убраны комментарии.
